e2e/process-services-cloud/process-custom-filters.e2e.ts

```diff
@@ -28,73 +28,102 @@ import { AppListCloudPage, LocalStorageUtil } from '@alfresco/adf-testing';
 import resources = require('../util/resources');
 
 import { browser, protractor } from 'protractor';
+import CONSTANTS = require('../util/constants');
+import { ProcessListCloudConfiguration } from './processListCloud.config';
+import { EditProcessFilterConfiguration } from './editProcessFilter.config';
 
 describe('Process list cloud', () => {
 
-    describe('Process List', () => {
-        const settingsPage = new SettingsPage();
-        const loginSSOPage = new LoginSSOPage();
-        const navigationBarPage = new NavigationBarPage();
-        const appListCloudComponent = new AppListCloudPage();
-        const processCloudDemoPage = new ProcessCloudDemoPage();
-        const tasksCloudDemoPage = new TasksCloudDemoPage();
-
-        let tasksService: TasksService;
-        let processDefinitionService: ProcessDefinitionsService;
-        let processInstancesService: ProcessInstancesService;
-        let queryService: QueryService;
-
-        let completedProcess, runningProcessInstance, switchProcessInstance, noOfApps;
-        const candidateuserapp = resources.ACTIVITI7_APPS.CANDIDATE_USER_APP.name;
-
-        beforeAll(async (done) => {
-            settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false);
-            loginSSOPage.clickOnSSOButton();
-            loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+    const settingsPage = new SettingsPage();
+    const loginSSOPage = new LoginSSOPage();
+    const navigationBarPage = new NavigationBarPage();
+    const appListCloudComponent = new AppListCloudPage();
+    const processCloudDemoPage = new ProcessCloudDemoPage();
+    const tasksCloudDemoPage = new TasksCloudDemoPage();
+
+    let tasksService: TasksService;
+    let processDefinitionService: ProcessDefinitionsService;
+    let processInstancesService: ProcessInstancesService;
+    let queryService: QueryService;
+    let activitiUser, activitiUserRoleId;
+
+    let completedProcess, runningProcessInstance, switchProcessInstance, noOfApps;
+    const candidateuserapp = resources.ACTIVITI7_APPS.CANDIDATE_USER_APP.name;
+
+    const apiService = new ApiService('activiti', TestConfig.adf.hostBPM, TestConfig.adf.hostSso, 'BPM');
+    const identityService = new IdentityService(apiService);
+    const rolesService = new RolesService(apiService);
+    const processListCloudConfiguration = new ProcessListCloudConfiguration();
+    const editProcessFilterConfiguration = new EditProcessFilterConfiguration();
+    let processListCloudConfigFile, editProcessFilterConfigFile;
+
+    beforeAll(async (done) => {
+        settingsPage.setProviderBpmSso(TestConfig.adf.hostBPM, TestConfig.adf.hostSso, TestConfig.adf.hostIdentity, false);
+        loginSSOPage.clickOnSSOButton();
+        loginSSOPage.loginSSOIdentityService(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
```

Comment:
remove the use of the admin user , create before a user  with identityService.createActivitiUserWithRole